src/Rings/PBWAlgebraQuo.jl

```diff
@@ -215,29 +278,38 @@ function (Q::PBWAlgQuo{T, S})(c::T) where {T, S}
 end
 
 function (Q::PBWAlgQuo)(c::IntegerUnion)
-  return PBWAlgQuoElem(Q, base_ring(Q)(c))
+  return PBWAlgQuoElem(Q, PBWAlgElem(base_ring(Q), Q.sring(c)))
 end
 
 function (Q::PBWAlgQuo)(a::PBWAlgQuoElem)
-  parent(a) == Q || error("coercion impossible")
-  return a
+    if (parent(a) != Q)
```

Comment:
```suggestion
    if parent(a) != Q
```
and similarly elsewhere. Well, or just use `@req` here